dockerfile/rocm4.0-pytorch1.7.0.dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM rocm/pytorch:rocm4.0_ubuntu18.04_py3.6_pytorch_1.7.0
+
+# OS:
+#   - Ubuntu: 18.04
+# AMD:
+#   - ROCm: 4.0
+#   - HIP: 3.21.2
+#   - MIOpen: 2.9.0
+#   - RCCL: 2.7.8
+# Mellanox:
+#   - OFED: 5.2-2.2.3.0
+
+LABEL maintainer="SuperBench"
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | sudo APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key add -
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    autoconf \
+    automake \
+    build-essential \
+    curl \
+    dmidecode \
+    git \
+    jq \
+    libaio-dev \
+    libcap2 \
+    libpci-dev \
+    libtinfo5 \
+    libtool \
+    lshw \
+    net-tools \
+    openssh-client \
+    openssh-server \
+    pciutils \
+    util-linux \
+    vim \
+    wget \
+    && \
+    apt-get autoremove && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/*
+
+# Configure SSH
+RUN mkdir -p /root/.ssh && \
+    touch /root/.ssh/authorized_keys && \
+    mkdir -p /var/run/sshd && \
+    sed -i "s/[# ]*PermitRootLogin prohibit-password/PermitRootLogin yes/" /etc/ssh/sshd_config && \
+    sed -i "s/[# ]*Port.*/Port 22/" /etc/ssh/sshd_config && \
+    echo "PermitUserEnvironment yes" >> /etc/ssh/sshd_config
+
+# Install OFED
+ENV OFED_VERSION=5.2-2.2.3.0
+RUN cd /tmp && \
+    wget -q http://content.mellanox.com/ofed/MLNX_OFED-${OFED_VERSION}/MLNX_OFED_LINUX-${OFED_VERSION}-ubuntu18.04-x86_64.tgz && \
+    tar xzf MLNX_OFED_LINUX-${OFED_VERSION}-ubuntu18.04-x86_64.tgz && \
+    PATH=/usr/bin:${PATH} MLNX_OFED_LINUX-${OFED_VERSION}-ubuntu18.04-x86_64/mlnxofedinstall --user-space-only --without-fw-update --force --all && \
+    rm -rf MLNX_OFED_LINUX-${OFED_VERSION}*
+
+ENV PATH="${PATH}" \
+    LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}" \
+    SB_HOME="/opt/superbench" \
+    SB_MICRO_PATH="/opt/superbench"
+
+WORKDIR ${SB_HOME}
+
+ADD third_party third_party
+RUN make -j -C third_party rocm
+
+ADD . .
+RUN python3 -m pip install . \
+    transformers==4.3.3 && \
```

Comment:
There have one issue we once mentioned. Use `python3 -m pip install [.torch]` will reinstall `pytorch` and `torchvision` even change the setup to  `'torch>=0.0.0'`.
Better use the fix version to avoid the potential incomparable issue of performance.

Reply:
if torch>=0.0.0 will install pytorch, then seems the image does not have a system wide pytorch? it may install pytorch in user path but other dependencies for superbench are installed in system path.
I think it's better to move user path pytorch to system path to resolve python's dependencies, otherwise if the image is run as other user pytorch will not be found

Reply:
setup will install all package into system path even the requirement is satisfied? from the log, it has detected the torch in user path:
`Requirement already satisfied: torch>=0.0.0 in /root/.local/lib/python3.6/site-packages (from superbench==0.2.1) (1.7.0a0)`

Reply:
By moving from user path to conda path:
`RUN mv /root/.local/bin/* /opt/conda/bin/ && mv /root/.local/lib/python3.6/site-packages/* /opt/conda/lib/python3.6/`

newer torch is still downloaded and failed to install:
```
Collecting torch>=0.0.0
  Downloading torch-1.9.0-cp36-cp36m-manylinux1_x86_64.whl (831.4 MB)
     |████████████████████████████████| 831.4 MB 41 kB/s

  Attempting uninstall: torch
    Found existing installation: torch 1.7.0a0
    Not uninstalling torch at /opt/conda/lib/python3.6, as it is in the standard library.
    Can't uninstall 'torch'. No files were found to uninstall.
```
Are there some solutions to let torch be detected and then avoid downloading?

Reply:
did you try `python3 -m pip install . --user`? if this doesn't reinstall pytorch, it should be path issue

Reply:
set `torch>=0.0.0` in setup.py and `python3 -m pip install .[torch] --user ` will reinstall pytorch.

Reply:
add the following workaround:
`RUN mv /root/.local/bin/* /opt/conda/bin/ && mv /root/.local/lib/python3.6/site-packages/* /opt/conda/lib/python3.6/`